.github/workflows/markdown_style.yml

```diff
@@ -14,4 +14,6 @@ jobs:
     - name: Install dependencies
       run: npm install `cat npm-requirements.txt`
     - name: Run style checks
+      # see: https://stackoverflow.com/a/39991116/5201771
+      # alternative (probably): npx remark ...
       run: npx remark src/*.md src/*/*.md --frail --rc-path .remarkrc
```

Comment:
locally I can also run `npm run env -- ` instead of npx, but somehow that fails on the CI